pkgs/os-specific/linux/sgxsdk/default.nix

```diff
@@ -0,0 +1,96 @@
+{ lib,
+  stdenvNoCC,
+  fetchurl,
+  fetchFromGitHub,
+  autoconf,
+  automake,
+  binutils,
+  bison,
+  cmake,
+  file,
+  flex,
+  gcc,
+  git,
+  gnumake,
+  gnum4,
+  libtool,
+  nasm,
+  ocaml,
+  ocamlPackages,
+  openssl,
+  perl,
+  python3,
+  texinfo
+}:
+
+stdenvNoCC.mkDerivation {
```

Comment:
why `stdenvNoCC`?

Reply:
Good question. Before starting to work on this I had no prior experience with `nix`, and my starting point was the work of Intel's linux-sgx project at https://github.com/intel/linux-sgx/blob/master/linux/reproducibility/Dockerfile#L21.

I tried to use _just_ `stdenv` but it was somehow failing. I'll give another try.

Reply:
If we need no compiler this is fine.